locked-asset/proxy_dex/src/wrapped_farm_attributes.rs

```diff
@@ -139,8 +139,7 @@ pub fn merge_wrapped_farm_tokens<M: CallTypeApi + StorageMapperApi>(
         )
     };
 
-    let merged_farm_tokens =
-        merge_farm_tokens_through_farm(original_caller, farm_address, farm_tokens_to_merge);
+    let merged_farm_tokens = merge_farm_tokens_through_farm(farm_address, farm_tokens_to_merge);
```

Comment:
In case of merge do we recompute the energy ? If not, we might have some issue.

Reply:
Nope, we do not recompute in case of plain merging.